### ISSUE TITLE
java.lang.NullPointerException and error 500 in /scm/api/v2/auth/access_token

### ISSUE THREAD

**Issue by fnajera-rac-de** [scm-manager/scm-manager#1771]
I am writing a script that runs in the same machine where SCM Manager is installed. 

Our SCM Manager is normally accessed via a reverse proxy, say https://name.tld
We have configured the option "Base URL" to https://name.tld and we have the option Force Base URL on (this could have something to do with the error, I guess).

When I try to get an access token using the URL, it works:

```
POST https://name.tld/scm/api/v2/auth/access_token with { "username":"existinguser", "password":"correctpassword", "cookie": false, "grant_type":"password" }
```

But when I used the internal URL (the rest is exactly the same), it blows up:

```
POST http://127.0.0.1:8080/scm/api/v2/auth/access_token with { "username":"existinguser", "password":"correctpassword", "cookie": false, "grant_type":"password" }
```

The work around is obviously to use the external URL, but maybe you want to handle this situation better.

Answer:
```
{"transactionId":"6ZSfxMpMF1M1k","errorCode":"CmR8GCJb31","context":[],"message":"internal server error"}
```

Log file:
<pre>
2021-08-13 11:01:26.933 [qtp1347137144-10254] [6ZSfxMpMF1M1k] WARN  sonia.scm.api.FallbackExceptionMapper - mapping unexpected java.lang.NullPointerException to status code
java.lang.NullPointerException: null
       at sonia.scm.api.v2.resources.AuthenticationResource.authenticate(AuthenticationResource.java:192)
       at sonia.scm.api.v2.resources.AuthenticationResource.authenticateViaJSONBody(AuthenticationResource.java:182)
       at jdk.internal.reflect.GeneratedMethodAccessor168.invoke(Unknown Source)
       at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
       at java.base/java.lang.reflect.Method.invoke(Method.java:566)
       at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:170)
       at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:130)
       at org.jboss.resteasy.core.ResourceMethodInvoker.internalInvokeOnTarget(ResourceMethodInvoker.java:646)
       at org.jboss.resteasy.core.ResourceMethodInvoker.invokeOnTargetAfterFilter(ResourceMethodInvoker.java:510)
       at org.jboss.resteasy.core.ResourceMethodInvoker.lambda$invokeOnTarget$2(ResourceMethodInvoker.java:460)
       at org.jboss.resteasy.core.interception.jaxrs.PreMatchContainerRequestContext.filter(PreMatchContainerRequestContext.java:364)
       at org.jboss.resteasy.core.ResourceMethodInvoker.invokeOnTarget(ResourceMethodInvoker.java:462)
       at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:420)
       at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:394)
       at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:69)
       at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:492)
       at org.jboss.resteasy.core.SynchronousDispatcher.lambda$invoke$4(SynchronousDispatcher.java:261)
       at org.jboss.resteasy.core.SynchronousDispatcher.lambda$preprocess$0(SynchronousDispatcher.java:161)
       at org.jboss.resteasy.core.interception.jaxrs.PreMatchContainerRequestContext.filter(PreMatchContainerRequestContext.java:364)
       at org.jboss.resteasy.core.SynchronousDispatcher.preprocess(SynchronousDispatcher.java:164)
       at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:247)
       at org.jboss.resteasy.plugins.server.servlet.ServletContainerDispatcher.service(ServletContainerDispatcher.java:249)
       at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:60)
       at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:55)
       at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
       at com.google.inject.servlet.ServletDefinition.doServiceImpl(ServletDefinition.java:290)
       at com.google.inject.servlet.ServletDefinition.doService(ServletDefinition.java:280)
       at com.google.inject.servlet.ServletDefinition.service(ServletDefinition.java:184)
       at com.google.inject.servlet.ManagedServletPipeline.service(ManagedServletPipeline.java:89)
       at com.google.inject.servlet.FilterChainInvocation.doFilter(FilterChainInvocation.java:85)
       at sonia.scm.legacy.RepositoryLegacyProtocolRedirectFilter$Worker.noRedirect(RepositoryLegacyProtocolRedirectFilter.java:127)
       at sonia.scm.legacy.RepositoryLegacyProtocolRedirectFilter$Worker.checkPathElements(RepositoryLegacyProtocolRedirectFilter.java:100)
       at sonia.scm.legacy.RepositoryLegacyProtocolRedirectFilter$Worker.doFilter(RepositoryLegacyProtocolRedirectFilter.java:89)
       at sonia.scm.legacy.RepositoryLegacyProtocolRedirectFilter.doFilter(RepositoryLegacyProtocolRedirectFilter.java:71)
       at sonia.scm.web.filter.HttpFilter.doFilter(HttpFilter.java:93)
       at com.google.inject.servlet.FilterChainInvocation.doFilter(FilterChainInvocation.java:82)
       at sonia.scm.web.security.TokenRefreshFilter.doFilter(TokenRefreshFilter.java:83)
       at sonia.scm.web.filter.HttpFilter.doFilter(HttpFilter.java:93)
       at com.google.inject.servlet.FilterChainInvocation.doFilter(FilterChainInvocation.java:82)
       at sonia.scm.filter.MDCFilter.doFilter(MDCFilter.java:103)
       at sonia.scm.web.filter.HttpFilter.doFilter(HttpFilter.java:93)
       at com.google.inject.servlet.FilterChainInvocation.doFilter(FilterChainInvocation.java:82)
       at sonia.scm.web.security.ApiAuthenticationFilter.doFilter(ApiAuthenticationFilter.java:95)
       at sonia.scm.web.filter.HttpFilter.doFilter(HttpFilter.java:93)
       at com.google.inject.servlet.FilterChainInvocation.doFilter(FilterChainInvocation.java:82)
       at sonia.scm.security.TokenExpiredFilter.doFilter(TokenExpiredFilter.java:66)
       at sonia.scm.web.filter.HttpFilter.doFilter(HttpFilter.java:93)
       at com.google.inject.servlet.FilterChainInvocation.doFilter(FilterChainInvocation.java:82)
       at sonia.scm.filter.BaseUrlFilter.doFilter(BaseUrlFilter.java:109)
       at sonia.scm.web.filter.HttpFilter.doFilter(HttpFilter.java:93)
       at com.google.inject.servlet.FilterChainInvocation.doFilter(FilterChainInvocation.java:82)
       at sonia.scm.metrics.HttpMetricsFilter.doFilter(HttpMetricsFilter.java:68)
       at sonia.scm.web.filter.HttpFilter.doFilter(HttpFilter.java:93)
       at com.google.inject.servlet.FilterChainInvocation.doFilter(FilterChainInvocation.java:82)
       at org.apache.shiro.guice.web.SimpleFilterChain.doFilter(SimpleFilterChain.java:44)
       at org.apache.shiro.web.servlet.AdviceFilter.executeChain(AdviceFilter.java:108)
       at org.apache.shiro.web.servlet.AdviceFilter.doFilterInternal(AdviceFilter.java:137)
       at org.apache.shiro.web.servlet.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:125)
       at org.apache.shiro.guice.web.SimpleFilterChain.doFilter(SimpleFilterChain.java:41)
       at org.apache.shiro.web.servlet.AdviceFilter.executeChain(AdviceFilter.java:108)
       at org.apache.shiro.web.servlet.AdviceFilter.doFilterInternal(AdviceFilter.java:137)
       at org.apache.shiro.web.servlet.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:125)
       at org.apache.shiro.guice.web.SimpleFilterChain.doFilter(SimpleFilterChain.java:41)
       at org.apache.shiro.web.servlet.AbstractShiroFilter.executeChain(AbstractShiroFilter.java:450)
       at org.apache.shiro.web.servlet.AbstractShiroFilter$1.call(AbstractShiroFilter.java:365)
       at org.apache.shiro.subject.support.SubjectCallable.doCall(SubjectCallable.java:90)
       at org.apache.shiro.subject.support.SubjectCallable.call(SubjectCallable.java:83)
       at org.apache.shiro.subject.support.DelegatingSubject.execute(DelegatingSubject.java:387)
       at org.apache.shiro.web.servlet.AbstractShiroFilter.doFilterInternal(AbstractShiroFilter.java:362)
       at org.apache.shiro.web.servlet.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:125)
       at com.google.inject.servlet.FilterChainInvocation.doFilter(FilterChainInvocation.java:82)
       at com.google.inject.servlet.ManagedFilterPipeline.dispatch(ManagedFilterPipeline.java:121)
       at com.google.inject.servlet.GuiceFilter.doFilter(GuiceFilter.java:133)
       at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:201)
       at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1601)
       at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:548)
       at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143)
       at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:602)
       at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
       at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:235)
       at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1624)
       at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:233)
       at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1435)
       at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:188)
       at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:501)
       at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1594)
       at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:186)
       at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1350)
       at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
       at org.eclipse.jetty.server.handler.HandlerCollection.handle(HandlerCollection.java:146)
       at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
       at org.eclipse.jetty.server.Server.handle(Server.java:516)
       at org.eclipse.jetty.server.HttpChannel.lambda$handle$1(HttpChannel.java:388)
       at org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:633)
       at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:380)
       at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:273)
       at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:311)
       at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:105)
       at org.eclipse.jetty.io.ChannelEndPoint$1.run(ChannelEndPoint.java:104)
       at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:336)
       at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:313)
       at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:171)
       at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:129)
       at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:375)
       at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:773)
       at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:905)
       at java.base/java.lang.Thread.run(Thread.java:829)
</pre>

**Comment by sdorra** [comment]
How do you send the post request? My guess is that your client send the post request, scm-manager answers the request with a redirect (because of the internal url, that is what force base url does), your client follows the redirect and sends another request but this time without the json body.

I mark this issue as bug, because we should do a better validation here. Thanks for the report.

**Comment by fnajera-rac-de** [comment]
TL;DR: the HTTP redirect code is technically wrong.

Thanks for the insight, it helped to look at the underlying problem.

Reading a bit further on the issue, it seems you should be returning HTTP 307 instead of HTTP 302 if the request is not GET or HEAD (see https://everything.curl.dev/http/redirects )

If you simulate this (using the flag --post302 in curl), it actually works.

Original request:

`$ curl --verbose --location --request POST "http://localhost:8080/scm/api/v2/auth/access_token" --header "Accept: */*" --header "Content-Type: application/json" --data-raw "{ \"username\":\"USER\", \"password\":\"PASS\", \"cookie\": false, \"grant_type\":\"password\" }"
`
<pre>
Note: Unnecessary use of -X or --request, POST is already inferred.
*   Trying 127.0.0.1:8080...
* TCP_NODELAY set
* Connected to localhost (127.0.0.1) port 8080 (#0)
> POST /scm/api/v2/auth/access_token HTTP/1.1
> Host: localhost:8080
> User-Agent: curl/7.68.0
> Accept: */*
> Content-Type: application/json
> Content-Length: 82
>
* upload completely sent off: 82 out of 82 bytes
* Mark bundle as not supporting multiuse
< HTTP/1.1 302 Found
< Date: Tue, 17 Aug 2021 09:05:40 GMT
< Location: https://SERVER/scm/api/v2/auth/access_token
< Content-Length: 0
< Server: Jetty(9.4.35.v20201120)
<
* Connection #0 to host localhost left intact
* Issue another request to this URL: 'https://SERVER/scm/api/v2/auth/access_token'
* Switch from POST to GET
*   Trying fd00:0:ac:ffce::100:127:443...
* TCP_NODELAY set
* Connected to SERVER (fd00:0:ac:ffce::100:127) port 443 (#1)
* ALPN, offering h2
* ALPN, offering http/1.1
* successfully set certificate verify locations:
*   CAfile: /etc/ssl/certs/ca-certificates.crt
  CApath: /etc/ssl/certs
* TLSv1.3 (OUT), TLS handshake, Client hello (1):
* TLSv1.3 (IN), TLS handshake, Server hello (2):
* TLSv1.3 (IN), TLS handshake, Encrypted Extensions (8):
* TLSv1.3 (IN), TLS handshake, Certificate (11):
* TLSv1.3 (IN), TLS handshake, CERT verify (15):
* TLSv1.3 (IN), TLS handshake, Finished (20):
* TLSv1.3 (OUT), TLS change cipher, Change cipher spec (1):
* TLSv1.3 (OUT), TLS handshake, Finished (20):
* SSL connection using TLSv1.3 / TLS_AES_256_GCM_SHA384
* ALPN, server accepted to use h2
* Server certificate:
...
*  SSL certificate verify ok.
* Using HTTP2, server supports multi-use
* Connection state changed (HTTP/2 confirmed)
* Copying HTTP/2 data in stream buffer to connection buffer after upgrade: len=0
* Using Stream ID: 1 (easy handle 0x55e04e8a0d40)
> POST /scm/api/v2/auth/access_token HTTP/2
> Host: SERVER
> user-agent: curl/7.68.0
> accept: */*
> content-type: application/json
>
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* old SSL session ID is stale, removing
* Connection state changed (MAX_CONCURRENT_STREAMS == 128)!
< HTTP/2 500
< server: nginx/1.18.0 (Ubuntu)
< date: Tue, 17 Aug 2021 09:05:41 GMT
< content-type: application/vnd.scmm-error+json;v=2
< content-length: 105
< cache-control: no-cache
<
* Connection #1 to host SERVER left intact
{"transactionId":"3VSgKlzo01vax","errorCode":"CmR8GCJb31","context":[],"message":"internal server error"}
</pre>

Request if you returned 307 instead of 302:

`$ curl --verbose --location --request POST "http://localhost:8080/scm/api/v2/auth/access_token" --header "Accept: */*" --header "Content-Type: application/json" --data-raw "{ \"username\":\"USER\", \"password\":\"PASS\", \"cookie\": false, \"grant_type\":\"password\" }" --post302
`
<pre>
Note: Unnecessary use of -X or --request, POST is already inferred.
*   Trying 127.0.0.1:8080...
* TCP_NODELAY set
* Connected to localhost (127.0.0.1) port 8080 (#0)
> POST /scm/api/v2/auth/access_token HTTP/1.1
> Host: localhost:8080
> User-Agent: curl/7.68.0
> Accept: */*
> Content-Type: application/json
> Content-Length: 82
>
* upload completely sent off: 82 out of 82 bytes
* Mark bundle as not supporting multiuse
< HTTP/1.1 302 Found
< Date: Tue, 17 Aug 2021 09:01:39 GMT
< Location: https://SERVER/scm/api/v2/auth/access_token
< Content-Length: 0
< Server: Jetty(9.4.35.v20201120)
<
* Connection #0 to host localhost left intact
* Issue another request to this URL: 'https://SERVER/scm/api/v2/auth/access_token'
*   Trying fd00:0:ac:ffce::100:127:443...
* TCP_NODELAY set
* Connected to SERVER (fd00:0:ac:ffce::100:127) port 443 (#1)
* ALPN, offering h2
* ALPN, offering http/1.1
* successfully set certificate verify locations:
*   CAfile: /etc/ssl/certs/ca-certificates.crt
  CApath: /etc/ssl/certs
* TLSv1.3 (OUT), TLS handshake, Client hello (1):
* TLSv1.3 (IN), TLS handshake, Server hello (2):
* TLSv1.3 (IN), TLS handshake, Encrypted Extensions (8):
* TLSv1.3 (IN), TLS handshake, Certificate (11):
* TLSv1.3 (IN), TLS handshake, CERT verify (15):
* TLSv1.3 (IN), TLS handshake, Finished (20):
* TLSv1.3 (OUT), TLS change cipher, Change cipher spec (1):
* TLSv1.3 (OUT), TLS handshake, Finished (20):
* SSL connection using TLSv1.3 / TLS_AES_256_GCM_SHA384
* ALPN, server accepted to use h2
* Server certificate:
...
*  SSL certificate verify ok.
* Using HTTP2, server supports multi-use
* Connection state changed (HTTP/2 confirmed)
* Copying HTTP/2 data in stream buffer to connection buffer after upgrade: len=0
* Using Stream ID: 1 (easy handle 0x56028deced40)
> POST /scm/api/v2/auth/access_token HTTP/2
> Host: SERVER
> user-agent: curl/7.68.0
> accept: */*
> content-type: application/json
> content-length: 82
>
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* old SSL session ID is stale, removing
* Connection state changed (MAX_CONCURRENT_STREAMS == 128)!
* We are completely uploaded and fine
< HTTP/2 401
< server: nginx/1.18.0 (Ubuntu)
< date: Tue, 17 Aug 2021 09:01:49 GMT
< cache-control: no-cache
<
* Connection #1 to host SERVER left intact
</pre>

Relevant diff:

```diff
--- 	Tue Aug 17 08:58:51 2021 UTC
+++ 	Tue Aug 17 08:58:51 2021 UTC
@@ -19,7 +19,6 @@
 <
 * Connection #0 to host localhost left intact
 * Issue another request to this URL: 'https://SERVER/scm/api/v2/auth/access_token'
-* Switch from POST to GET
 *   Trying fd00:0:ac:ffce::100:127:443...
 * TCP_NODELAY set
 * Connected to SERVER (fd00:0:ac:ffce::100:127) port 443 (#1)
@@ -50,17 +49,16 @@
 > user-agent: curl/7.68.0
 > accept: */*
 > content-type: application/json
+> content-length: 82
 >
 * TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
 * TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
 * old SSL session ID is stale, removing
 * Connection state changed (MAX_CONCURRENT_STREAMS == 128)!
-< HTTP/2 500
+* We are completely uploaded and fine
+< HTTP/2 401
 < server: nginx/1.18.0 (Ubuntu)
 < date: Tue, 17 Aug 2021 ...
-< content-type: application/vnd.scmm-error+json;v=2
-< content-length: 105
 < cache-control: no-cache
 <
-* Connection #1 to host SERVER left intact
-{"transactionId":"3VSgKlzo01vax","errorCode":"CmR8GCJb31","context":[],"message":"internal server error"}
+* Connection #1 to host SERVER left intact
```

I'd suggest that, apart from improving the validation, you consider whether the redirect code can be changed to prevent this issue with standard-compliant HTTP clients.

**Comment by sdorra** [comment]
I can confirm that the status code 307 is the better choice for forcing the base url. We will fix this with next version see #1778. 

But i can't reproduce the NullPointerException. Even if the redirect is done with a get request, the status code which returned from the `access_token` endpoint is 405, because the endpoint does not support get requests.

**Comment by fnajera-rac-de** [comment]
> But i can't reproduce the NullPointerException.

I can make it happen here just by using this (note: it is a POST, not a get):

<pre>
curl --verbose --location --request POST "http://localhost:8080/scm/api/v2/auth/access_token" --header "Accept: */*" --header "Content-Type: application/json" --data-raw "{ \"username\":\"USER\", \"password\":\"PASS\", \"cookie\": false, \"grant_type\":\"password\" }"
</pre>

<pre>
fd00:0:ac:ffce::100:127 - - [23/Aug/2021:09:40:14 +0200] "POST /scm/api/v2/auth/access_token HTTP/2.0" 500 105 "-" "curl/7.68.0"
2021-08-23 09:40:14.503 [qtp1347137144-8449] [8vSgtVdfx15kR] WARN  sonia.scm.api.FallbackExceptionMapper - mapping unexpected java.lang.NullPointerException to status code 500
java.lang.NullPointerException: null
       at sonia.scm.api.v2.resources.AuthenticationResource.authenticate(AuthenticationResource.java:192)
       at sonia.scm.api.v2.resources.AuthenticationResource.authenticateViaJSONBody(AuthenticationResource.java:182)
       at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
       at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
       ...
</pre>

Is there any log setting I can change to help you diagnose this?

**Comment by fnajera-rac-de** [comment]
@sdorra I saw the ticket got closed. Should I open a new one with my last reply?

**Comment by sdorra** [comment]
Sorry @fnajera-rac-de i think this was a mistake.

**Comment by sdorra** [comment]
Ok, i was able to reproduce the `NullPointerException`. The `NullPointerException` occurs only with the json endpoint and not with the form data endpoint. However with the fix from #1778 the `NullPointerException` seems to be fixed.

**Comment by sdorra** [comment]
We could provide you a test package? Which package do you use?

**Comment by fnajera-rac-de** [comment]
We use Ubuntu 20.04.3 LTS. But since you were able to reproduce it and test the fix, there's no need for an extra test release for us. It's not a pressing issue anyway - it's pretty much an edge case that probably nobody else will ever encounter. We can wait for the next release and test it again on our side then.

**Comment by sdorra** [comment]
Thanks for reporting the fix will be available with the upcoming version 2.23.0
